styleguide/source/assets/scss/01-atoms/_search-field.scss

```diff
@@ -35,9 +35,9 @@
     @extend .icon--search;
     background-color: transparent;
     background-size: 25px 30px;
-    background-position: 0 0;
-    width: 25px;
-    height: 30px;
+    background-position: center;
+    width: 25px !important;
```

Comment:
I would avoid using important. Can we use specific styles to target this instead?